.github/workflows/ci.yml

```diff
@@ -119,7 +119,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ruby-2.7]
+        ruby: [ruby-2.7, ruby-3.0]
```

Comment:
Are you intentionally running rubocop on two versions of ruby?

Reply:
for the moment yes, will soon remove 2.7.